### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by torn4dom4n**

### CODE REVIEW

**Comment by sarah11918** [comment]
Ah, thank you! This content did just get moved into one of our reference pages, so I'll update to that link.  :)

File: src/pages/en/getting-started.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ See examples of some of the key concepts and patterns of an Astro site!
 
 📚 Read more about Astro’s [project structure.](/en/core-concepts/project-structure)
 
-📚 Learn more about Astro’s [built-in components.](/en/reference/builtin-components)
+📚 Learn more about Astro’s [components.](/en/core-concepts/astro-components)
```
Comment (sarah11918) [inline src/pages/en/getting-started.md:61]:
```suggestion
📚 Learn more about Astro’s [built-in components](/en/reference/api-reference/#built-in-components).
```

This is where that information moved to!

File: src/pages/en/getting-started.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ See examples of some of the key concepts and patterns of an Astro site!
 
 📚 Read more about Astro’s [project structure.](/en/core-concepts/project-structure)
```
Comment (sarah11918) [inline src/pages/en/getting-started.md:59]:
```suggestion
📚 Read more about Astro’s [project structure](/en/core-concepts/project-structure).
```

File: src/pages/en/getting-started.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ See examples of some of the key concepts and patterns of an Astro site!
...
 
-📚 Learn more about Astro’s [built-in components.](/en/reference/builtin-components)
+📚 Learn more about Astro’s [components.](/en/core-concepts/astro-components)
 
 📚 Explore Astro’s [API.](/en/reference/api-reference)
```
Comment (sarah11918) [inline src/pages/en/getting-started.md:63]:
```suggestion
📚 Explore Astro’s [API](/en/reference/api-reference).
```

**Comment by sarah11918** [comment]
@torn4dom4n Thanks again! Can you please make the changes above?  (We DO have built-in components, just as part of a different page now.) And, if you're fixing it, I would also love to have the periods at the end of the sentence not inside the links.  😄

**Comment by torn4dom4n** [comment]
> @torn4dom4n Thanks again! Can you please make the changes above? (We DO have built-in components, just as part of a different page now.) And, if you're fixing it, I would also love to have the periods at the end of the sentence not inside the links. 😄
Absolutely.

**Comment by sarah11918** [comment]
Perfect! Thank you for your contribution.  :)

**Pull request merged**